Circle.cpp

```diff
@@ -1,14 +1,33 @@
 #ifndef Circle_cpp
 #define Circle_cpp
+
+#define _USE_MATH_DEFINES
+
 #include <iostream>
-#include "Draw.cpp"
+#include <math.h>
+
+#include "Shape.cpp"
+#include "Area.cpp"
+#include "Input.cpp"
+
 using namespace std;
-class Circle : public Draw
+class Circle : public Shape , public Input
 {
-public:
-    void drawShape() override
+    private:
+        int radius = inputRadius ();
```

Comment:
Ideally, we should not be invoking a function in the declaration since this causes it to be invoked immediately on Object creation. And any required changes in the behaviour will result in changes in the class implementation.